packages/params/test/unit/__snapshots__/forkName.test.ts.snap

```diff
@@ -0,0 +1,41 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
```

Comment:
can't we achieve the same with simple unit tests?

Reply:
That's simple unit tests. 

> Snapshot tests are a very useful tool whenever you want to make sure the output of your functions does not change unexpectedly.

https://vitest.dev/guide/snapshot.html

Reply:
I don't see how this more readable, the first test can just be like this
```ts
it("should have valid allForks", () => {
    expect(forkAll).toEqual(["phase0", "altair", "bellatrix", "capella", "deneb"]);
  });
```

From the docs, ti feels to me that snapshosts are useful if
- function outputs a large object (e.g. beacon block) and you wanna just hard code it here
- values change over time and hard to modify, could use [updating-snapshots](https://vitest.dev/guide/snapshot.html#updating-snapshots) to update test case

The only value that changes over time here is `forkAll` which is simple to update the test case and faster than running something like `vitest -u`.

I think we should avoid adding new test patterns unless those are useful

Reply:
I think a good addition of tests for the whole type refactor would be to add type tests like we have for metrics
https://github.com/ChainSafe/lodestar/blob/8cb08ff56f66022c84d8ca1a91e82dba290e644a/packages/utils/test/types/metrics.test-d.ts#L5 as of right now it's really only tested in the src code that the types work as we expect them to

Reply:
> function outputs a large object (e.g. beacon block) and you wanna just hard code it here

There is no rule how much large an object be to test inline or snapshot. So in general if we just need to test a static values then snapshots are the good use there.

Reply:
> The only value that changes over time here is forkAll which is simple to update the test case and faster than running something like vitest -u.

Seems you had not done the snapshot testing before, otherwise there is no faster or slower thing to compare for a adding `-u` one flag. Once you used to it, you will understand it's benefits and simplicity much better.

Reply:
> I think a good addition of tests for the whole type refactor would be to add type tests like we have for metrics

There is explicit and integral usage of those types everywhere, so if something wrong the code will not build. So I don't see value in adding a types tests. 

If the types we had were complex accepting multiple generics, manipulating based on generic conditions with different paths, then to test all possible outcome we should have added type tests.

Reply:
As I understand `-u` will update your snapshot to match what the code outputs, I haven't used it, that's just from the docs

Reply:
don't see a specific reason why snapshots are better than simple unit tests, having everything in a one place but I guess you just wanted to try out snapshots, we can keep it as is

Reply:
Not wanting to try out, rather been using snapshot testing for many years. It's much better way to test the static values.